src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -125,16 +129,13 @@ public KubernetesSlave(PodTemplate template, String nodeDescription, String clou
     protected KubernetesSlave(String name, @Nonnull PodTemplate template, String nodeDescription, String cloudName, String labelStr,
                            ComputerLauncher computerLauncher, RetentionStrategy rs)
             throws Descriptor.FormException, IOException {
-        super(name,
-                nodeDescription,
-                template.getRemoteFs(),
-                1,
-                template.getNodeUsageMode() != null ? template.getNodeUsageMode() : Node.Mode.NORMAL,
-                labelStr,
-                computerLauncher,
-                rs,
-                template.getNodeProperties());
-
+        super(name, null, computerLauncher);
+        setNodeDescription(nodeDescription);
+        setNumExecutors(1);
+        setMode(template.getNodeUsageMode() != null ? template.getNodeUsageMode() : Node.Mode.NORMAL);
+        setLabelString(labelStr);
+        setRetentionStrategy(rs);
+        setNodeProperties(template.getNodeProperties());
```

Comment:
Is this related, or just a deprecation warning?

Reply:
I guess you addressed the deprecation while dealing with the `remoteFs`?

Reply:
Yes